gulpfile.js

```diff
@@ -15,14 +15,24 @@ const gulp = require('gulp');
 const rename = require('gulp-rename');
 const cp = require('child_process');
 
-const serverName = 'com.redhat.microprofile.ls-uber.jar';
+const microprofileServerName = 'com.redhat.microprofile.ls-uber.jar';
 const extensions = ['com.redhat.microprofile.jdt.core', 'com.redhat.microprofile.jdt.quarkus'];
-const serverDir = '../quarkus-ls/microprofile.ls/com.redhat.microprofile.ls';
+const microprofileServerDir = '../quarkus-ls/microprofile.ls/com.redhat.microprofile.ls';
 const extensionDir = '../quarkus-ls/microprofile.jdt';
 
-gulp.task('buildServer', (done) => {
-  cp.execSync(mvnw() + ' clean verify -DskipTests', { cwd: serverDir , stdio: 'inherit' });
-  gulp.src(serverDir + '/target/' + serverName)
+const quarkusServerExtName = 'com.redhat.quarkus.ls-0.0.1-SNAPSHOT.jar';
```

Comment:
You can't hard code the version, better use glob patterns to catch the jar, regardless of its version